code/game/machinery/kitchen/gibber.dm

```diff
@@ -295,18 +248,42 @@ obj/machinery/gibber/New()
 		src.operating = 0
 		update_icon()
 
-/obj/machinery/gibber/proc/startautogibbing(mob/living/victim as mob)
-	if(src.operating)
+
+
+//auto-gibs anything that bumps into it
+/obj/machinery/gibber/autogibber
+	name = "autogibber"
+	desc = "Keep far, far away."
+	icon_state = "autogibber"
+
+/obj/machinery/gibber/autogibber/New()
```

Comment:
Why not, as long as it calls the parent? I see it done a ton of places in the code.